frontend/src/lib/components/RightPanel/rightPanel.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
+	import Post from './RightPost.svelte';
+</script>
+
+<div class="fixed right-0 h-screen w-80">
```

Comment:
not `fixed`, make a wrapper for right panel.
well check how reddit is doing this.